index.html

```diff
@@ -222,7 +222,83 @@ <h3 class="work-description">
                     </li>
                 </ul>
                 <div class="action">
-                    <a href="#" class="action-btn btn"><span>See Project</span></a>
+                    <a href="#" data-target="#work-modal" class="action-btn btn"><span>See Project</span></a>
+                </div>
+            </div>
+        </div>
+        <!-- Modal -->
+        <div class="popup" id="work-modal">
+            <div class="modal">
+                <div class="modal-header">
+                    <div>
+                        <h2>Tonic</h2>
+                        <div class="frame">
+                            <div class="client">
+                                <h3>Canopy</h3>
+                            </div>
+                            <div class="counter1"></div>
+                            <div class="role">
+                                <h3>Back End Dev</h3>
+                            </div>
+                            <div class="counter2"></div>
+                            <div class="year">
+                                <h3>2015</h3>
+                            </div>
+                        </div>
+                    </div>
+                    <img src="./asset/images/close-icon-dark.png" height="14" width="14" class="modal-close"
+                        alt="Close Modal Icon">
+                </div>
+                <div class="modal-body">
+                    <img class="desktop-img" src="./asset/images/Modal-Image-Desktop.png" alt="Modal Image">
```

Comment:
- [ ] Kindly fix the modal image in the desktop version as it currently looks strectched when compared to the figma design.

Your design
![image](https://user-images.githubusercontent.com/81932478/179822479-77389e80-40c6-4011-b6f5-c094f2ecf1d7.png)

Provided design
![image](https://user-images.githubusercontent.com/81932478/179822749-b512b19e-b7c2-4f9e-ada2-ec3fe304fb02.png)

Reply:
Image height corrected. Thanks